frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/shared/NodeChecker/node-checker.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import { Popover, PopoverPosition, Text, TextVariants, Button } from '@patternfly/react-core';
+import {
+  LoadingInline,
+  ResourceLink,
+  ExternalLink,
+  resourcePath,
+} from '@console/internal/components/utils';
+import { NodeKind } from '@console/internal/module/k8s';
+import { getName } from '@console/shared';
+import './node-checker.scss';
+
+export const NodeChecker: React.FC<NodeCheckerProps> = ({ qualifiedNodes, isLoading = false }) => {
+  const size = qualifiedNodes.length;
+  const buttonText = size === 1 ? '1 Node' : `${size} Nodes`;
+  const icon = size > 0 ? <CheckCircleIcon /> : <ExclamationCircleIcon />;
+  return (
+    <div className="kv-node-checker__container">
+      <Popover
+        headerContent={<div>{buttonText} found</div>}
+        position={PopoverPosition.right}
+        className="kv-node-checker__popover"
+        bodyContent={qualifiedNodes.map((node) => (
+          <ExternalLink
+            key={getName(node)}
+            href={resourcePath('Node', getName(node))}
+            text={<ResourceLink linkTo={false} kind="Node" name={getName(node)} />}
+          />
+        ))}
+      >
+        <div>
+          {isLoading ? (
+            <LoadingInline />
+          ) : (
+            <Button
+              className="kv-node-checker__popover-btn"
+              isDisabled={isLoading || size === 0}
+              variant="link"
+              icon={icon}
+            >
+              <Text component={TextVariants.h4}>{buttonText}</Text>
+            </Button>
+          )}
+        </div>
+      </Popover>
+      <Text component={TextVariants.h4}>
+        {isLoading ? ' Searching for Nodes' : ' qualify for this node selector'}
```

Comment:
Before the users clicks add label we shouldn't see this message. There is no selector yet. I would suggest saying "Add a Node selector label to see qualifying nodes" 
Also are you going to use PF inline notifications here? I Think the blue/grey is problematic especially without a timeout. You don't really notice it changing.